pom.xml

```diff
@@ -3,7 +3,7 @@
 
     <groupId>org.openmrs.module</groupId>
     <artifactId>fhir2</artifactId>
-    <version>1.2.0-SNAPSHOT</version>
+    <version>1.2.0-IP-SNAPSHOT</version>
```

Comment:
Is it possible to leave out these version changes from this PR? That way I can actually merge this 😁